### ISSUE TITLE
Text modifiers are broken when reaching the end of line

### ISSUE THREAD

**Issue by savioret** [rospdf/pdf-php#173]
I just updated to latest version

**I confirm this breaks in changeset #e2c5edc**

The thing is now when a text modifier (bold, italic, underline) breaks the line before closing the tag callback, it does not close it, and also truncates the text. Seems like if the remaining block of text after the line break is just not processed and keeps open forever.

![image](https://user-images.githubusercontent.com/5497332/233345794-56d70aae-06b6-41f7-891f-247ebfacc417.png)

Quick example:
```php
        ext_require('pdf-php/Cezpdf.php');
        $pdf = new \Cezpdf('a4', 'portrait');
        $pdf->ez['fontSize'] = 11;
        $pdf->ezText("adasdsadas ads sdaasd sda dsa saddsdsa sdadas sda sda sdasda  <b>XXXX YYYY ZZZZ TTTTT HHHH GGGG MORE BOLD</b> bla bla bla bla");
        $pdf->ezText("This text should not be bold\n");

        $pdf->ezText("adasdsadas ads sdaasd sda dsa saddsdsa sdadas sda sda sdasda  <i>XXXX YYYY ZZZZ TTTTT HHHH GGGG MORE BOLD</i> bla bla bla bla");
        $pdf->ezText("This text should not be italic (nor bold)");
        
        $pdf->ezStream();
```

**Comment by ole1986** [comment]
I think we have to revert the previous change asap.
Bad try!

**Comment by ole1986** [comment]
Fixed with PR #174
